src/Core/Tests/Unit/Input/AssumeRoleRequestTest.php

```diff
@@ -11,31 +11,49 @@ class AssumeRoleRequestTest extends TestCase
 {
     public function testRequestBody(): void
     {
-        self::markTestIncomplete('Not implemented');
-
         $input = new AssumeRoleRequest([
-            'RoleArn' => 'change me',
-            'RoleSessionName' => 'change me',
-            'PolicyArns' => [new PolicyDescriptorType([
-                'arn' => 'change me',
-            ])],
-            'Policy' => 'change me',
-            'DurationSeconds' => 1337,
+            'RoleArn' => 'arn:aws::iam::123456789012:role/demo',
+            'RoleSessionName' => 'John-session',
+            'PolicyArns' => [
+                new PolicyDescriptorType(['arn' => 'arn:aws:iam::123456789012:policy/demopolicy1']),
+                new PolicyDescriptorType(['arn' => 'arn:aws:iam::123456789012:policy/demopolicy2']),
+            ],
+            'Policy' => '{"Version":"2012-10-17","Statement":[{"Sid": "Stmt1","Effect": "Allow","Action": "s3:*","Resource": "*"}]}',
+            'DurationSeconds' => 1800,
             'Tags' => [new Tag([
-                'Key' => 'change me',
-                'Value' => 'change me',
+                'Key' => 'Project',
+                'Value' => 'Pegasus',
+            ]), new Tag([
+                'Key' => 'Team',
+                'Value' => 'Engineering',
+            ]), new Tag([
+                'Key' => 'Cost-Center',
+                'Value' => '12345',
             ])],
-            'TransitiveTagKeys' => ['change me'],
-            'ExternalId' => 'change me',
-            'SerialNumber' => 'change me',
-            'TokenCode' => 'change me',
+            'TransitiveTagKeys' => ['Project', 'Cost-Center'],
+            'ExternalId' => '123ABC',
         ]);
 
+        /** @see https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html */
```

Comment:
I think such comment is helpfull to understand where the "expectation" come from

Reply:
👍 

I added data before but that was from request I made myself.